src/core/lv_obj.c

```diff
@@ -231,6 +231,8 @@ lv_obj_t * lv_obj_create(lv_obj_t * parent)
 void lv_obj_add_flag(lv_obj_t * obj, lv_obj_flag_t f)
 {
     LV_ASSERT_OBJ(obj, MY_CLASS);
+    if((obj->flags & f) == f) /*Check if all flags are set*/
```

Comment:
Use `lv_obj_has_flag`

Reply:
This is a great idea! lv_obj_has_flag function does exactly that!